lib/tlitest/test_tlog.py

```diff
@@ -2,48 +2,56 @@
 
 import os
 import ast
+import sys
 import stat
 import time
-import pytest
-import pexpect
+import socket
 import inspect
-from pexpect import pxssh
-from systemd import journal
+from shutil import copyfile
 from tempfile import mkdtemp
+from systemd import journal
+
+import pexpect
+from pexpect import pxssh
 
+import pytest
 
 def journal_find_last():
+    """ helper to get the last journal entry """
     j = journal.Reader()
     j.seek_tail()
     entry = j.get_previous()
-    while('TLOG_REC' not in entry):
+    while 'TLOG_REC' not in entry:
         entry = j.get_previous()
     return entry
 
 
 def check_journal(pattern):
+    """ helper to check last journal entry contains pattern """
     time.sleep(1)
     entry = journal_find_last()
     message = entry['MESSAGE']
     out_txt = ast.literal_eval(message)['out_txt']
-    if pattern in out_txt:
-        return 0
-    else:
-        return 1
+    retval = 0
+    if pattern not in out_txt:
+        retval = 1
+    return retval
```

Comment:
You can condense the above four lines into one:

    return pattern not in out_txt

Regardless of how you do it, I see you're returning the result of the comparison, but nothing checks for it, so no test would fail, no matter which result you get. Is that your intention?

The above concerns `check_outfile` as well.

Reply:
Latest commit added changes to how check_journal and check_output work.   IIRC my original intention was to assert but that wasn't being done in the tests.  Looking at how it's used, I've instead moved the assert I originally intended into each function.  I also found failures in tests just today that I didn't see before so I changed the functions to loop a few times and check again.